content/300-guides/050-database/100-developing-with-prisma-migrate/180-generating-down-migrations.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Generating down migrations
+metaTitle: Generating down migrations
+metaDescription: How to generate down migrations
+tocDepth: 2
+---
+
+<TopBlock>
+
+This guide describes how to generate a down migration SQL file that reverses a given [migration file](/concepts/components/prisma-migrate#migration-history).
+
+</TopBlock>
+
+## About down migrations
+
+When generating a migration SQL file, you may wish to also create a "down migration" SQL file that reverses the schema changes in the corresponding "up migration" file.
+
+This guide explains how to use Prisma Migrate's [`migrate diff` command](/reference/api-reference/command-reference#migrate-diff) to create a down migration, and how to apply it to your production database with the [`db execute`](/reference/api-reference/command-reference#db-execute) command.
+
+<Admonition type="info">
+
+This guide applies to generating SQL down migrations for relational databases only. It does not apply to MongoDB.
+
+</Admonition>
+
+<Admonition type="info">
+
+The `migrate diff` and `db execute` commands are available in Preview in versions `3.9.0` and later, and are generally available in versions `3.13.0` and later.
+
+</Admonition>
+
+## Considerations when generating down migrations
```

Comment:
Wouldn't it be useful to mention here already that the down migrations needs to be created before the up migration is applied?